.github/workflows/scripts/update_manifest.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
+
 set -o nounset
 set -o errexit
 
 
-readonly TRAVIS_PULL_REQUEST="${TRAVIS_PULL_REQUEST:-}"
-readonly TRAVIS_BRANCH="${TRAVIS_BRANCH:-}"
+readonly GITHUB_PULL_REQUEST="${GITHUB_PULL_REQUEST:-}"
+readonly GITHUB_BRANCH="${GITHUB_BRANCH:-}"
```

Comment:
Github variables are different,
these are some of the translations I tried: https://github.com/fao89/pulp_file/blob/6066/.github/workflows/ci.yaml#L47-L60

Ref: https://docs.github.com/en/actions/reference/environment-variables

Reply:
I exported those variables as part of the job 

```yaml
  env:
    GITHUB_PULL_REQUEST: 873
    GITHUB_BRANCH: ci_template_test
    GITHUB_REPO_SLUG: ansible/galaxy_ng
```

Those vars are present, the error is happening because file misses `+x` permission.